packages/components/src/components/rating-stars/rating-stars.tsx

```diff
@@ -141,37 +191,52 @@ export class RatingStars {
         this.precision
       ),
       0,
-      this.numOfStars
+      this.max
     );
     return star;
   }
 
+  getValueFromXCoordinate(coordinate: number) {
+    const containerLeft = this.element.getBoundingClientRect().left;
+    const containerWidth = this.element.getBoundingClientRect().width;
+    return clamp(
+      this.roundToPrecision(
+        ((coordinate - containerLeft) / containerWidth) * this.max,
+        this.precision
+      ),
+      0,
+      this.max
+    );
+  }
+
   roundToPrecision(numberToRound: number, precision = 1) {
     const multiplier = 1 / precision;
     return Math.ceil(numberToRound * multiplier) / multiplier;
   }
 
   getAriaLabel() {
-    return this.ariaTranslation
-      .replace(/\$\{x\}/gi, this.rating.toString())
-      .replace(/\$\{y\}/gi, this.numOfStars.toString());
+    return this.ariaLabelTranslation
+      .replace(/\$\{x\}/gi, this.value.toString())
+      .replace(/\$\{y\}/gi, this.max.toString());
```

Comment:
I would use `value` and `max` here intead of `x` and `x`, like: `${value} out of ${max} stars`, since it's more explicit.

Reply:
...done